project/Dependencies.scala

```diff
@@ -28,7 +28,6 @@ object Dependencies {
   // needs to be inline with the aeron version, check
   // https://github.com/real-logic/aeron/blob/1.x.y/build.gradle
   val agronaVersion = "1.19.2"
-  val nettyVersion = "3.10.6.Final"
   val netty4Version = "4.1.96.Final"
```

Comment:
Since we are no longer using netty3, maybe we can just alias `netty4Version` to `nettyVersion`?